doc/contribute/modifying_contributions.rst

```diff
@@ -0,0 +1,67 @@
+.. _modifying_contributions:
+
+Modifying Contributions made by other developers
+************************************************
+
+Scenarios
+#########
+
+In general, Zephyr contributors and collaborators are encouraged to assist
+as reviewers in pull requests, so that patches may be approved and merged
+to Zephyr's main branch as part of the original pull requests. The authors
+of the pull requests are responsible for amending their original commits
+following the review process.
+
+There are occasions, however, when a contributor might need to modify patches
+included in pull requests that are submitted by other Zephyr contributors.
+For instance, this is the case when
```

Comment:
Generally paragraphs that precede a bullet list in rst text in Zephyr end with either `:` or `::`.  Having neither doesn't seem right.

Reply:
Agreed. Fixed this now